Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -20,6 +22,27 @@ public class ReportService : IReportService
         public JiraRequestContext JiraRequestContext { get; set; }
 
         #region IReportService Implementation
+        public List<DashboardItem> GetDashboardData(long instanceId)
+        {
+            var users = GetAllAtlassianUsers(instanceId);
+            var usersIds = users.Select(p => p.Id).ToList();
+            var worklogs = GetLastWorklogsByUsers(usersIds, instanceId);
```

Comment:
I think we overcomplicate things like this. Taking the user id's, then take the worklogs for all user, the interate throught the users and add the worklogs.
I think it would be better to have the method GetLastWorklogsByUsers -> GetLastWorklogsByUser; and when you iterate through the users, call this method.

Reply:
I didn't want this, because it will make a db call for each user and it will be really inefficient. This way, I'm scaling - no matter the number of users, the call will take just as long.